core/trino-main/src/main/java/io/trino/operator/OperatorInfo.java

```diff
@@ -15,10 +15,10 @@
 
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
-import io.trino.operator.partition.PartitionedOutputOperator.PartitionedOutputInfo;
 import io.trino.operator.TableWriterOperator.TableWriterInfo;
 import io.trino.operator.exchange.LocalExchangeBufferInfo;
 import io.trino.operator.join.JoinOperatorInfo;
+import io.trino.operator.partition.PartitionedOutputOperator.PartitionedOutputInfo;
```

Comment:
This should probably o into the previous commit

Reply:
fixed